### PR TITLE
Load schema in Tableau based on initial catalog

### PR DESCRIPTION
**Pull request by chrisunder**

### CODE REVIEW

File: presto-main/src/main/resources/webapp/tableau/presto-connector.html [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
         function populateCatalogs() {
             var catalogClient = new StatementClient({
-                    'catalog': 'hive',
```
Comment (electrum) [inline presto-main/src/main/resources/webapp/tableau/presto-connector.html]: Presto supports `SHOW CATALOGS` without specifying a catalog or schema. We shouldn't have to specify these here (though that will require changes in the JS `StatementClient`).

Comment (chrisunder) [inline presto-main/src/main/resources/webapp/tableau/presto-connector.html]: Schema and catalog have been removed

**Comment by electrum** [comment]
The commit message title is too long. Please limit to ~50 characters. See this guide for details: http://chris.beams.io/posts/git-commit/

File: presto-main/src/main/resources/webapp/tableau/presto-client.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ function StatementClient(connectionData, headerCallback, dataCallback, errorCall
     this.headers = {
         "X-Presto-User": this.user ? this.user : 'N/A',
         "X-Presto-Source": this.source,
-        "X-Presto-Catalog": this.catalog,
-        "X-Presto-Schema": this.schema,
+        "X-Presto-Catalog": this.catalog ? this.catalog : 'N/A',
```
Comment (martint) [inline presto-main/src/main/resources/webapp/tableau/presto-client.js]: `N/A` is not a proper value to pass via the X-Presto-Catalog header. If anything, these headers should be omitted when the catalog and schema are unavailable.

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**